openedx/core/djangoapps/oauth_dispatch/management/commands/edx_clear_expired_tokens.py

```diff
@@ -33,6 +34,7 @@ def clear_table_data(self, query_set, batch_size, model):
             batch_ids = qs.values_list('id', flat=True)
             with transaction.atomic():
                 model.objects.filter(pk__in=list(batch_ids)).delete()
+                sleep(1)
```

Comment:
See how this is done in https://github.com/edx/ecommerce/blob/master/ecommerce/core/management/commands/clean_history.py.